app/scripts/lib/account-tracker.js

```diff
@@ -347,7 +354,10 @@ export default class AccountTracker {
    * @param {Array} addresses - An array of hex addresses to stop tracking.
    */
   removeAccounts(addresses) {
-    const { accounts, accountsByChainId } = this.store.getState();
+    const { accounts: _accounts, accountsByChainId: _accountsByChainId } =
```

Comment:
What are your thoughts on moving this to a separate PR? This seems unrelated to switching to `#` syntax for private methods.

Reply:
Done. Let's get this merged first then https://github.com/MetaMask/metamask-extension/pull/22306